beacon-chain/core/state/transition.go

```diff
@@ -227,8 +323,11 @@ func ProcessBlock(
 //    process_crosslinks(state)
 //    process_rewards_and_penalties(state)
 //    process_registry_updates(state)
+//    # @process_reveal_deadlines
```

Comment:
?

Reply:
That's what's written in the python code of the spec, I think it means it's a place holder for a phase 1 function. Want me to remove it?

Reply:
yeah, it's a comment within a comment lol